services/tenant-ui/frontend/src/store/governanceStore.ts

```diff
@@ -8,20 +8,91 @@ export const useGovernanceStore = defineStore('governance', () => {
   const schemaTemplates: any = ref(null);
   const selectedSchemaTemplate: any = ref(null);
   const schemaTemplateFilters: any = ref(null);
+  
+  const credentialTemplates: any = ref(null);
+  const selectedCredentialTemplate: any = ref(null);
+  const credentialTemplateFilters: any = ref(null);
 
   const loading: any = ref(false);
   const error: any = ref(null);
 
   // getters
+  const schemaLabelValue = (item: any) => {
+    let result = null;
+    if (item != null) {
+      result = { 
+        'label': `${item.name} (${item.version})`, 
+        'value': item.schema_template_id,
+        'status': item.status,
+        'can_issue': item.credential_templates.length > 0
+      }
+    }
+    return result;
+  };
+
+  const schemaTemplateDropdown = computed(() => {
+    //transform list of schemas into label / value pair
+    let result: any[] = [];
+    if (schemaTemplates.value != null) {
+      result = schemaTemplates.value
+        .map((item: any) => schemaLabelValue(item));
+    }
+    return result;
+  });
+
+  const credDefLabelValue = (item: any) => {
+    let result = null;
+    if (item != null) {
+      result = { 
+        'label': `${item.name}`, 
+        'value': item.credential_template_id,
+        'status': item.status
+      }
+    }
+    return result;
+  };
+
+  const credentialTemplateDropdown = computed(() => {
+    //transform list of credential templates into label / value pair
+    let result: any[] = [];
+    if (credentialTemplates.value != null) {
+      result = credentialTemplates.value
+        .map((item: any) => credDefLabelValue(item));
+    }
+    return result;
+  });
 
+  
   // actions
 
   // grab the tenant api
   const tenantApi = useTenantApi();
 
   async function listSchemaTemplates() {
     selectedSchemaTemplate.value = null;
-    return fetchList('/tenant/v1/governance/schema_templates/', schemaTemplates, error, loading);
+    schemaTemplates.value = null;
+    //return fetchList('/tenant/v1/governance/schema_templates/', schemaTemplates, error, loading);
+    
+    // this is a total hack for expedience, lets get cred templates at the same time...
+    // just so we can show it all in one grid, this should happen at the api level...
+    const schemas = ref();
+    await fetchList('/tenant/v1/governance/schema_templates/', schemas, error, loading);
+    await listCredentialTemplates();
+    schemaTemplates.value = await Promise.all(schemas.value.map(async (item: any) => {
+      // find any credential templates...
+      const templates = ref();
+      await fetchList('/tenant/v1/governance/credential_templates/', templates, error, loading, {schema_template_id: item.schema_template_id});
+      return {
+        ...item,
+        credential_templates: templates
+      };
+    }));
+    return schemaTemplates.value;
+  }
```

Comment:
Yeah hack for expedience so we can iterate on this but yeah I dont think we'd want to do the cred def fetch in the map since that's looping that API call for each schema then. Could do the 2 lists _first_ and then map them together by ID, but like you said this might be a job for the API